pkg/search.go

```diff
@@ -70,7 +70,6 @@ func DoSearch(stores []storetypes.KubeconfigStore, config *types.Config, stateDi
 
 	resultChannel := make(chan DiscoveredContext)
 	wgResultChannel := sync.WaitGroup{}
-	wgResultChannel.Add(len(stores))
```

Comment:
Now that we don't have the wait counter set up here any more, can we still call `waitGroup.done()` [here](https://github.com/danielfoehrKn/kubeswitch/blob/27c05669596f068bee65c6763a77bc6b7e9369c2/pkg/search.go#L107) when we read from the index?  Looks to me like we wouldn't have a corresponding `Add()` any more.

Reply:
I'm not sure to understand where to .done() ^^